README.md

```diff
@@ -226,22 +83,22 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
Just as the previous reviewer mentioned, please remove any trace of template that is not personalized from your ReadMe. This comment was intended as guide for you on what to add to this section but it has no relevance other users interacting with you project. That is why it is good to always delete lines like this in your ReadMes.

Reply:
Traces of the template removed